### ISSUE TITLE
Initial Admin user not created because auth.adminUser creates the wrong env name for keycloak

### ISSUE THREAD

**Issue by rezuma** [bitnami/charts#11193]
### Name and Version

CHART NAME: keycloak CHART VERSION: 9.4.0 APP VERSION: 18.0.2

### What steps will reproduce the bug?

auth.adminUser creates the env var in the container KEYCLOAK_ADMIN_USER but the env name that Keycloak is expecting is KEYCLOAK_ADMIN, as a result the chart doesnt add the initial admin user.
A way around this is not use auth values from the chart and use instead extraEnvs
```

extraEnvVars:
  - name: KEYCLOAK_ADMIN
    value: "admin"
  - name: KEYCLOAK_ADMIN_PASSWORD
    value: whateverpasswordyoulike


```

### Are you using any custom parameters or values?

```
 auth:
   adminUser: user
   adminPassword: "dsadasdas"
   managementUser: manager
   managementPassword: "asdasdasdasd"
```

```
extraEnvVars:
  - name: KEYCLOAK_ADMIN
    value: "admin"
  - name: KEYCLOAK_ADMIN_PASSWORD
    value: ds,jfsdkjfsd
```

### What do you see instead?

It creates the wrong Env Var in the container

**Comment by rafariossaa** [comment]
Using the latest chart version (9.4.1) I can see that keycloak is run with the right environment variables.

```
$ helm install mykey bitnami/keycloak --set auth.adminUser=user1 --set auth.adminPassword=test1
...
$ kubectl exec -it mykey-keycloak-0 bash
$ cat /proc/1/environ | tr '\0' '\n' | grep ADMIN
KEYCLOAK_ADMIN_PASSWORD=test1
KEYCLOAK_ADMIN_USER=user1
KEYCLOAK_ADMIN=user1
```

The `KEYCLOAK_ADMIN_USER` is copied onto `KEYCLOAK_ADMIN` [in the container scripts](https://github.com/bitnami/bitnami-docker-keycloak/blob/542c5ad4d59f940927a6ea6896deea26cf65dfb2/18/debian-11/rootfs/opt/bitnami/scripts/keycloak-env.sh#L98).

Could you check your versions an if the process has the right environment variables ?

**Comment by patrickbeam** [comment]
Ran into this same issue today. 

TLDR, there is a PR waiting to be merged that should solve this problem. https://github.com/bitnami/charts/pull/11497

I was planning to open the same PR after I edited `configmap-env-vars.yaml`

To correct this issue just edit that file and replace `KEYCLOAK_ADMIN_USER` with `KEYCLOAK_ADMIN`
